exercises/saddle-points/SaddlePoints.cs

```diff
@@ -4,11 +4,7 @@
 
 public class SaddlePoints
```

Comment:
Done that. Something is still fishy about the generator. When I run `dotnet run -e saddle-points` it regenerates on the output the same as I would run `dotnet run -e SaddlePoints` (SaddlePointsTests.cs). VS asks if I want to reload the file, because change happened. Tho nothing changes. Git doesn't show change neither.